classes/ImageManager.php

```
@@ -23,6 +23,25 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
+if (version_compare(PHP_VERSION, '8.0.0', '<')) {
```

Instead of creating a fake class, we should ignore errors related to GdImage as we don't directly use this class

---

Approved with @PierreRambaud